notebook/notebook/templates/workshop/index.html

```diff
@@ -3,13 +3,6 @@
 {% block content %}
   <div class="vcentered">
     <h1>Welcome!</h1>
-    {% if userdata %}
-      <p>Welcome to the Hail-powered <b>{{ userdata['workshop_name'] }}</b> workshop!</p>
-    {% else %}
-      <p>Welcome to the Hail workshop service!  <a href="{{ base_path }}/login">Log
-	  in</a> to join the workshop.</p>
-    {% endif %}
```

Comment:
Any specific reason for objecting to this?  I liked saying Hail here.

Reply:
Tim and I talked it over. Seemed like too much text, which makes it more difficult to find the link I'm really on this page to find. If I'm logged in, I get the workshop name, but I already knew I was here for the workshop. If I'm not, I get a login link, but I don't need it, the notebook page already redirects to login.

Reply:
I'll update to keep Hail.

Reply:
Let me know if you like better